api/orchestrator/orchestrator.proto

```diff
@@ -107,6 +107,10 @@ service Orchestrator {
         get : "/v1/orchestrator/cloud_services/{filtered_cloud_service_id}/"
               "assessment_results"
       }
+      additional_bindings {
```

Comment:
I think we don't need additional bindings here, this should automatically be part of the query parameters, correct, @lebogg?

Reply:
Ok, that makes sense. But why is the additional_binding needed for filtered_cloud_service_id?

Reply:
I think its not needed but we liked the idea of associating it to the `/cloud_services` sub-resource path as well, but its not needed.

Reply:
Ok, that is confusing.

Reply:
I think we can remove all additional bindings and make all of the filtered_ properties optional then.

Reply:
Just done :)